src/controller/ProjectManager.py

```diff
@@ -365,4 +382,13 @@ def import_raw_data(self, path: str):
         :param path: path where the raw_data is located
         """
         raw_data = DataFrame(FileManager.import_(path))
+        raw_data = raw_data.set_index(raw_data.columns[0])
         self.get_project().set_raw_data(raw_data, path)
+
+    def export_raw_data(self, path: str):
+        """
+        Exports the raw_data
+        :param path: path where the raw_data is exported to
+        """
+        raw_data = self.get_project().get_raw_data()
```

Comment:
```suggestion
        raw_data = self.get_project().get_raw_data(with_derivatives=True)
```
Derivatives have to be added as this is currently only called when pressing `Export Data`.

Reply:
I tried this solution before (only changing get_raw_data() to get_raw_data(True))
See issue #80 for further details
It didn’t work